closure/goog/events/browserevent.js

```diff
@@ -223,8 +223,19 @@ goog.events.BrowserEvent.IEButtonMap = [
 goog.events.BrowserEvent.prototype.init = function(e, opt_currentTarget) {
   var type = this.type = e.type;
 
+  /** @type {Touch} */
+  var relevantTouch = null;
+  if (type == goog.events.EventType.TOUCHSTART ||
+      type == goog.events.EventType.TOUCHMOVE) {
+    relevantTouch = e.targetTouches[0];
```

Comment:
@mtsgrd this example is based on my `targettouches` branch: http://erilem.net/ngeo/layerorder-touch/layerorder.html. Does it work for you in iOS 8?